test/ae_database/AESetup.inc

```diff
@@ -0,0 +1,251 @@
+<?php
+function AEConnect($ae){
+    $databaseName = 'AEDemo';
+    $uid = 'yourUsername';
+    $pwd = 'yourPassword';
+    $server = 'yourServer';
+    $ae_str = "Disabled";
+    if ($ae)
+        $ae_str = "Enabled";
+    $connectionInfo = array("Database"=>$databaseName, "UID"=>$uid, "PWD"=>$pwd,
+                            "ReturnDatesAsStrings"=>true, "CharacterSet"=>"UTF-8", "ColumnEncryption"=>$ae_str);
+    $conn = sqlsrv_connect($server, $connectionInfo);
+    sqlsrv_assert($conn === false, "Could not connect.\n");
+    return ($conn);
+}
+
+function DropAETable($conn){
+    $sql_drop = "IF OBJECT_ID('dbo.test_AE', 'U') IS NOT NULL DROP TABLE [dbo].[test_AE]";
+    $stmt = sqlsrv_query($conn, $sql_drop);
+    sqlsrv_assert($stmt === false, "Error in dropping table.\n");
+}
```

Comment:
Perhaps think of another function name rather than sqlsrv_assert()? This sounds like one of the existing sqlsrv api...